packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -268,16 +246,55 @@ export class Account {
    * @param {string} from
    * @param {BalanceTransferPayload} payload
    * @param {string} payload[address]
-   * @param {AccountToAccountOptions} [options]
+   * @param {BalanceTransferAccountOptions} [options]
    * @param {UTXO[]} [options.utxos = []]
    * @param {string} [options.utxos.txid]
    * @param {number} [options.utxos.vout]
    * @return {Promise<string>}
    */
-  async accountToAccount (from: string, payload: BalanceTransferPayload, options: AccountToAccountOptions = { utxos: [] }): Promise<string> {
+  async accountToAccount (from: string, payload: BalanceTransferPayload, options: BalanceTransferAccountOptions = { utxos: [] }): Promise<string> {
     return await this.client.call('accounttoaccount', [from, payload, options.utxos], 'number')
   }
 
+  /**
+   * Create an Account to UXTOS transaction submitted to a connected node.
+   * Optionally, specific UTXOs to spend to create that transaction.
+   *
+   * @param {string} from
+   * @param {BalanceTransferPayload} payload
+   * @param {string} payload[address]
+   * @param {BalanceTransferAccountOptions} [options]
+   * @param {UTXO[]} [options.utxos = []]
+   * @param {string} [options.utxos.txid]
+   * @param {number} [options.utxos.vout]
+   * @return {Promise<string>}
+   */
+  async accountToUtxos (from: string, payload: BalanceTransferPayload, options: BalanceTransferAccountOptions = { utxos: [] }): Promise<string> {
```

Comment:
can you move it back to previous position??

Reply:
<img width="572" alt="Screenshot 2021-06-01 at 11 20 14 AM" src="https://user-images.githubusercontent.com/7101806/120262699-c938ec80-c2cc-11eb-85c5-e931e5c4b525.png">
<img width="360" alt="Screenshot 2021-06-01 at 11 24 20 AM" src="https://user-images.githubusercontent.com/7101806/120262709-cccc7380-c2cc-11eb-8105-4e46140039c3.png">

I just arrange the RPC according to this order

Reply:
I really appreciate the organizing effort, but I guess coming from a reviewer POV it is hards to see the changes going into. Some mental work involved. You might want to create separate PRs for refactoring in the future (please do if you see any).